docs/Rootless.md

```diff
@@ -0,0 +1,57 @@
+
+With this new target location, *all* projects will have to be recompiled to reference resources from and install to `/var/jb`. This involves changing the internal structure of .deb's to include this new path and specifying library and framework install paths using `@rpath` instead of absolute paths. Theos will handle both of these changes for you. For more information on what `@rpath` is and how it works, see Mike Ash's [informative blog post](http://www.mikeash.com/pyblog/friday-qa-2009-11-06-linking-and-install-names.html).
+
+### Implementation
+
+---
+
+**Important Notes**:
+- You must run `make clean` when switching between a rootful and rootless build
+
+- If you maintain a library, you'll want to add some variation of the following to your code so that it will have the correct install path:
+```make
+ifeq ($(THEOS_PACKAGE_SCHEME),rootless)
+XXX_LDFLAGS += -install_name @rpath/<lib-name>.dylib
+endif
+```
+
+- Likewise, if you maintain a framework, you'll want to add some variation of the following to your code so that it will have the correct install path:
+```make
+ifeq ($(THEOS_PACKAGE_SCHEME),rootless)
+XXX_LDFLAGS += -install_name @rpath/<project-name>.framework/<project-name>
+endif
+```
+
+- The iOS 14 arm64e ABI mentioned in [arm64e Deployment](arm64e-Deployment.html) is now *required* for the relevant devices
+
+---
+
+Theos supports building for the rootless scheme in a few ways:
+- Provides rootless-compatible libraries and frameworks in `$THEOS_VENDOR_LIBRARY_PATH/iphone/rootless`
+
+- Provides `rootless.h` -- a header that contains convenient macros to easily convert rootful paths to rootless ones in your code at compile-time, assuming you compile for the rootless scheme (see below)
+    - `#import <rootless.h>`
+    - Courtesy of [opa334](https://github.com/theos/headers/commit/9f00c9663aff892b512f87666dbfbf8fe4943e84)
+
+- `THEOS_PACKAGE_SCHEME=rootless` -- a variable to enable a handful of internal changes including:
+    - Searching for libraries and frameworks when linking in `$THEOS_LIBRARY_PATH/iphone/rootless` and `$THEOS_VENDOR_LIBRARY_PATH/iphone/rootless`
+    - Passing the relevant prefixed rpaths to the linker so your project can find the linked rootless libraries and frameworks on-device
+    - Sharing the install prefix (`THEOS_PACKAGE_INSTALL_PREFIX=/var/jb`) with the compiler for use in your code
+    - Setting the package architecture to `iphoneos-arm64` if your control file specifies `iphoneos-arm`
+
+Additional notes:
+- You do *not* need to create separate package identifiers for rootful/rootless versions of the same package
+    - Package managers will present only the compatible version to users
```

Comment:
Maybe it's worth noting that this is only true for the latest versions of modern/actively maintained package managers.

* Cydia: duplicates tweaks that have both rootful and rootless versions, but both lead to the rootful version so just a visual bug
* Sileo 2.3 and below: often has issues being unable to install versions of tweaks that exist in both rootful and rootless
* Saily (archived, and no one should really be using this anyway): seems to prioritize the rootless package if it exists, even on rootful
* Zebra: I think it's fine, but it doesn't hide packages with incompatible architectures and just warns about the incompatibility when actually trying to install them
* Installer: ???

Reply:
I was debating leaving it as-is but you're right ... with such inconsistent behavior across the board, it's definitely worth clarifying what is *likely* to work as stated.